src/shared/conf-parser.h

```diff
@@ -278,6 +278,7 @@ CONFIG_PARSER_PROTOTYPE(config_parse_uint32_invert_flag);
 CONFIG_PARSER_PROTOTYPE(config_parse_id128);
 CONFIG_PARSER_PROTOTYPE(config_parse_tristate);
 CONFIG_PARSER_PROTOTYPE(config_parse_string);
+CONFIG_PARSER_PROTOTYPE(config_parse_refuse_record_types);
```

Comment:
Please move this to resolved-conf.h

Reply:
@yuwata This was some leftover code which i forgot to move it over.  this is moved.